docusaurus-docs/conda-store/references/faq.md

```diff
@@ -115,29 +115,60 @@ c.CondaStore.build_key_version = 2
 ## Long paths on Windows
 
 conda-store supports Windows in standalone mode. However, when creating
-environments with certain packages, you may see errors like
+environments with certain packages, you may see errors like:
 
 ```bash
 ERROR:root:[WinError 206] The filename or extension is too long: 'C:\\...'
 ```
 
 This error is due to the fact that Windows has a limitation that file paths
-cannot be more than 260 characters. The fix is to set the registry key
+cannot be more than 260 characters.
+
+See [conda-store issue #588][max-path-issue] for more details.
+
+### Solution 1: Extended-length path prefix (`\\?\`)
```

Comment:
I thought solution (2) is required to enable solution (1) too? Maybe I am misremembering, though.

Reply:
Nope! That's the whole point of using the extended-length path prefix. Note how I set `LongPathsEnabled=0` in the CI job, too.